content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst

```diff
@@ -0,0 +1,160 @@
+
+   With these options selected, the :guilabel:`Production Analysis` report shows a bar chart for the
+   current year, with one bar for each product, signifying the average operation cost for one unit
+   of the product.
+
+   With this data, Mike is able to see that the average operation cost for the moon shoes is almost
+   twice the cost of the pogo stick. Using this insight, Tommy's Toys decides to cease production of
+   moon shoes, thus lowering their average cost of manufacturing operations.
+
+   .. image:: production_analysis/use-case.png
+      :align: center
+      :alt: The bar chart comparing the operation costs of the pogo stick and moon shoes.
+
+Use case: compare time periods
+==============================
+
+The :guilabel:`Production Analysis` report can also be used to compare data for two different time
+periods. This is accomplished using the options in the :guilabel:`Comparison` section of the
+:guilabel:`Search...` bar.
+
+.. example::
+   Furniture company *Fanny's Furnishings* wants to compare their production costs for the first and
+   second quarters of 2024, to see which products they spent the most money producing in each
+   quarter.
+
+   To compare the two time periods, shop floor supervisor Adam opens the
+   :menuselection:`Manufacturing` app, and navigates to the :guilabel:`Production Analysis` page. He
+   begins by selecting the :icon:`fa-pie-chart` :guilabel:`(pie chart)` graph type option at the top
+   of the page.
+
+   .. important::
+      The :guilabel:`Comparison` feature is meant to be used with the :icon:`fa-pie-chart`
+      :guilabel:`(pie chart)` graph type, or the :icon:`oi-view-pivot` :guilabel:`(pivot)` view.
+
+      A :guilabel:`Comparison` option can still be selected with the other view types enabled, but
+      doing so does not change the way data is displayed on the report.
+
+   Next, Adam selects the :guilabel:`Total Cost` option from the :guilabel:`Measures` drop-down
+   menu. This option displays the total amount spent producing each product.
+
+   In the :guilabel:`Search...` bar drop-down menu, he leaves the :guilabel:`2024` filter enabled in
+   the :guilabel:`End Date` section, and enables the :guilabel:`Q2` filter as well. With both of
+   these time periods selected, the pie chart shows data for the second quarter of 2024.
+
+   Finally, Adam selects the :guilabel:`End Date: Previous Period` option in the
+   :guilabel:`Comparison` section of the :guilabel:`Search...` bar. Doing so causes the pie chart to
+   be split into an inner circle, and an outer ring.
+
+   The outer ring shows data for the selected time period, quarter two of 2024. The inner circle
+   shows data for the previous time period, quarter one of 2024.
+
+   .. note::
+      If :guilabel:`End Date: Previous Year` is selected instead of :guilabel:`End Date:
+      Previous Period`, the inner circle shows data for the selected time period, one *year*
+      previous.
```

Comment:
early line break
```suggestion
      If :guilabel:`End Date: Previous Year` is selected instead of :guilabel:`End Date: Previous
      Period`, the inner circle shows data for the selected time period, one *year* previous.
```